client/app/components/gmap.jsx

```diff
@@ -35,7 +35,7 @@ class Gmap extends React.Component {
 
   overlayHeatmap() {
     var heatmapPoints = this.props.heatmapData.map(function(crime) {
-      return new google.maps.LatLng(crime.location.coordinates[0] - 0, crime.location.coordinates[1] - 0);
+      return new google.maps.LatLng(crime.location.coordinates[1] - 0, crime.location.coordinates[0] - 0);
```

Comment:
It's because, in order to perform geospatial queries, we have to store the point coordinates as long / lat, while google require lat / long. The storeAllLocations method in server/models/crime is updated, which necessitates updating these coords here. While we are not currently using the points for any particular geospatial queries, but are using the 'box' field in the db schema, I left the updated structure in case we (or someone later) finds a use case for querying the crime locations by point (for example, the number of crimes within a certain distance of a given point on the map).